yb-voyager/src/utils/commonVariables.go

```diff
@@ -78,11 +78,11 @@ type SchemaReport struct {
 }
 
 type SchemaSummary struct {
-	DBName     string     `json:"DbName,omitempty"`
-	SchemaName string     `json:"SchemaName,omitempty"`
-	DBVersion  string     `json:"DbVersion,omitempty"`
-	Notes      []string   `json:"Notes,omitempty"`
-	DBObjects  []DBObject `json:"DatabaseObjects"`
+	DBName      string     `json:"DbName,omitempty"`
+	SchemaNames string     `json:"SchemaName(s),omitempty"`
```

Comment:
i was expecting you to use `[]string` here. Since that will be a better representation, and also will be better for outside json parser(unaware of voyager logic) such that they will know it can contain multiple schemas not just one.